modules/apps/change-tracking/change-tracking-rest-impl/rest-openapi.yaml

```diff
@@ -588,6 +588,38 @@ paths:
                                     $ref: "#/components/schemas/CTEntry"
                                 type: array
             tags: ["CTEntry"]
+    "/ct-collections/{ctCollectionId}/ct-entries/by-model-class-name-id/{modelClassNameId}/by-model-class-pk/{modelClassPK}":
+        get:
+            operationId: getCtCollectionCTEntryByModelClassNameByModelClassPkModelClassPK
```

Comment:
There's two `ModelClassPK` in the operationId. Is this generated? If you leave it out, I believe it will be auto-generated.

Reply:
I tried leaving it out but still 2 ModelClassPKs were added when I ran rest builder

Reply:
I originally named the operand `getCtCollectionCTEntryByModelClassNameIdByModelClassPk`

Reply:
Did it get override by the rest builder? I thought if you set it manually, it will not be changed. Maybe try `PK` instead of `Pk`?

Reply:
It still gets overridden

Reply:
Ok, just leave it blank then and let rest builder generate it

Reply:
Leaving the operationId completely blank resulted in an error
`Task :apps:change-tracking:change-tracking-rest-impl:buildREST FAILED
Aug 01, 2024 9:42:03 AM com.liferay.portal.kernel.log.Jdk14LogImpl error
SEVERE: Invalid YAML in 'string', line 594, column 13:
                operationId: getCtCollectionCTEn ... 
                ^: found duplicate key operationId
com.liferay.portal.tools.rest.builder.internal.yaml.exception.InvalidYAMLException: Invalid YAML in 'string', line 594, column 13:
                operationId: getCtCollectionCTEn ... 
                ^: found duplicate key operationId
	at com.liferay.portal.tools.rest.builder.internal.yaml.YAMLUtil.loadOpenAPIYAML(YAMLUtil.java:50)
	at com.liferay.portal.tools.rest.builder.RESTBuilder._loadOpenAPIYAML(RESTBuilder.java:1846)
	at com.liferay.portal.tools.rest.builder.RESTBuilder._fixOpenAPIContentApplicationXML(RESTBuilder.java:1181)
	at com.liferay.portal.tools.rest.builder.RESTBuilder._checkOpenAPIYAMLFile(RESTBuilder.java:471)
	at com.liferay.portal.tools.rest.builder.RESTBuilder.build(RESTBuilder.java:201)
	at com.liferay.portal.tools.rest.builder.RESTBuilder.main(RESTBuilder.java:108)
Caused by: while constructing a mapping
 in 'string', line 593, column 13:
                operationId:
                ^
found duplicate key operationId
 in 'string', line 594, column 13:
                operationId: getCtCollectionCTEn ... 
                ^
`

Reply:
Remove the attribute entirely.

Reply:
I am still getting the same operationId `getCtCollectionCTEntryByModelClassNameByModelClassPkModelClassPK`
Is that ok?

Reply:
Yes, that's fine. Leave it as is.